pulsar-cli-utils/src/main/java/org/apache/pulsar/cli/ValueValidationUtil.java

```diff
@@ -27,31 +26,31 @@ public class ValueValidationUtil {
 
     public static void maxValueCheck(String paramName, long value, long maxValue) {
         if (value > maxValue) {
-            throw new ParameterException(paramName + " cannot be bigger than <" + maxValue + ">!");
+            throw new IllegalArgumentException(paramName + " cannot be bigger than <" + maxValue + ">!");
```

Comment:
Could we keep this pr only related to picocli.
we can create a new pr for this kind of change.

Reply:
To eliminate the jcommander dependency, this must be done.

ParameterException was provided by the jcommander.